tests/test_delish_2.py

```diff
@@ -0,0 +1,100 @@
+# mypy: allow-untyped-defs
+
+from recipe_scrapers._grouping_utils import IngredientGroup
+from recipe_scrapers.delish import Delish
+from tests import ScraperTest
+
+
+class TestDelishScraper(ScraperTest):
+
+    scraper_class = Delish
+    test_file_name = "delish_2"
```

Comment:
Nitpick: rename the existing test filename to `_1`?

Reply:
Added the _1 to the original test file and added test_file_name.